tests/contrib/django/test_django.py

```diff
@@ -1,3 +1,4 @@
+# -*- coding: utf-8 -*-
```

Comment:
can we have regression tests for the redis integrations too? if not then I think we should hold off updating the redis integrations in this PR.